shallow_backup.py

```diff
@@ -391,44 +449,47 @@ def read_config(config_path, config):
 
 
 # custom help options
-@click.command(context_settings=dict(help_option_names=['-h', '-help']))
+@click.command(context_settings=dict(help_option_names=['-h', '-help', '--help']))
 @click.option('-complete', is_flag=True, default=False, help="Back up everything.")
 @click.option('-dotfiles', is_flag=True, default=False, help="Back up dotfiles.")
+@click.option('-configs', is_flag=True, default=False, help="Back up app config files.")
 @click.option('-fonts', is_flag=True, default=False, help="Back up installed fonts.")
 @click.option('-packages', is_flag=True, default=False, help="Back up package libraries and installed applications.")
 @click.option('-old_path', is_flag=True, default=False, help="Skip setting new back up directory path.")
 @click.option('--new_path', default="DEFAULT", help="Input a new back up directory path.")
-@click.option('-reinstall', is_flag=True, default=False, help="Reinstall packages from package lists.")
+@click.option('-reinstall_packages', is_flag=True, default=False, help="Reinstall packages from package lists.")
+@click.option('-reinstall_configs', is_flag=True, default=False, help="Reinstall configs from configs backup.")
 @click.option('-delete_config', is_flag=True, default=False, help="Remove config file.")
 @click.option('-v', is_flag=True, default=False, help='Display version and author information and exit.')
-def cli(complete, dotfiles, packages, fonts, old_path, new_path, reinstall, delete_config, v):
+def cli(complete, dotfiles, configs, packages, fonts, old_path, new_path, reinstall_packages, reinstall_configs,
+		delete_config, v):
 	"""
 	Easily back up installed packages, dotfiles, and more.
 	"""
 
-	config_path = os.path.join(expanduser("~"), ".shallow-backup")
+	backup_config_path = os.path.join(expanduser("~"), ".shallow-backup")
```

Comment:
let's use the `_home_prefix()` function here instead since we have the function.